task/06-conditions-n-loops-tasks.js

```diff
@@ -226,7 +285,13 @@ function reverseString(str) {
  *   34143 => 34143
  */
 function reverseInteger(num) {
-    throw new Error('Not implemented');
+    let resultReversInteger = 0;
```

Comment:
For this task, you can also use a simpler solution through the "reverse" method, and converting to string and back.

Reply:
Fixed.